avocado/utils/vmimage.py

```diff
@@ -135,9 +135,23 @@ def get_image_url(self):
                                      "Wrong arch?" % (image, url_images))
 
 
-class FedoraImageProvider(ImageProviderBase):
+class FedoraImageProviderBase(ImageProviderBase):
     """
-    Fedora Image Provider
+    Base Fedora Image Provider
+    """
+
+    def get_image_url(self):
+        if int(self.version) >= 28:
+            cloud = 'Cloud'
+        else:
+            cloud = 'CloudImages'
+        self.url_images = self.url_images % cloud
+        return super(FedoraImageProviderBase, self).get_image_url()
+
+
+class FedoraImageProvider(FedoraImageProviderBase):
+    """
+    Base Fedora Image Provider
```

Comment:
can this docstring will be   `Fedora Image Provider`

Reply:
True, good catch!